<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveWindowClass</class>
 <widget class="QMainWindow" name="SaveWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>282</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>648</width>
    <height>282</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>648</width>
    <height>282</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>SaveWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="csvRadioButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>95</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>95</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CSV</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="jsonRadioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>95</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>95</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>JSON</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yamlRadioButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>95</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>95</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>YAML</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sqlRadioButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>95</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>95</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SQL</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="descAddValue">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>431</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>431</width>
      <height>41</height>
     </size>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Comic Sans MS','cursive','sans-serif'; font-size:10pt;&quot;&gt;Insert the file name below and select the extension&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveFileNameText">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1441</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="saveStatus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>251</width>
      <height>41</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSaveButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>93</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>93</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
